src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -66,10 +66,6 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('collectors.xml');
```

Comment:
@HeahDude apparently we can't do that. See https://github.com/symfony/symfony/pull/18934#discussion_r65502752